src/libcore/convert.rs

```diff
@@ -6,8 +6,10 @@
 //! - Implement the [`AsRef`] trait for cheap reference-to-reference conversions
 //! - Implement the [`AsMut`] trait for cheap mutable-to-mutable conversions
 //! - Implement the [`From`] trait for consuming value-to-value conversions
-//! - Implement the [`Into`] trait for consuming value-to-value conversions to types outside the current crate
-//! - The [`TryFrom`] and [`TryInto`] traits behave like [`From`] and [`Into`], but should be implemented when
+//! - Implement the [`Into`] trait for consuming value-to-value conversions to types
+//! outside the current crate
```

Comment:
```rust
//! - Implement the [`Into`] trait for consuming value-to-value conversions to types
//!   outside
```

Reply:
@lu-zero Thanks for your response! I will go and fix this (Y)

Reply:
Should be fixed now!